mino/minogrpc/mod.go

```diff
@@ -190,6 +199,7 @@ func NewMinogrpc(listen net.Addr, public *url.URL, router router.Router, opts ..
 		certs:  certs.NewInMemoryStore(),
 		curve:  elliptic.P521(),
 		random: rand.Reader,
+		noTLS:  false,
```

Comment:
double negation ( *no* TLS = *false* ) is a bit of a cognitive burden, especially with all the checks.
Out of compassion for my postprandial brain, I'd put this as `useTLS: true` and change the `DisableTLS` option to set it to false 😅